keymaps/intellij-idea-keymap.cson

```diff
@@ -230,6 +230,12 @@
   # Duplicate current line or selected block
   'ctrl-d': 'editor:duplicate-lines'
 
+  # Start new line
+  'shift-enter': 'editor:newline-below'
```

Comment:
What is the difference between `newline` and `newline-below` command? I tested it shortly but I don't see any difference. And `shift-enter` is already bound to `newline` in the default atom keymap.